Source/Mp3/Mp3FileReader.swift

```diff
@@ -8,12 +8,73 @@
 import Foundation
 
 class Mp3FileReader {
-    func readFrom(path: String) throws -> Data {
+    private let tagSizeParser: TagSizeParser
+    private let id3TagConfiguration: ID3TagConfiguration
+
+    init(tagSizeParser: TagSizeParser,
+         id3TagConfiguration: ID3TagConfiguration) {
+        self.tagSizeParser = tagSizeParser
+        self.id3TagConfiguration = id3TagConfiguration
+    }
+
+    /**
+      Read the entire mp3 file at path
+
+      - parameter path: the path to the mp3 file
+
+      - returns: mp3 data of the file
+
+      - throws: Could throw `InvalidFileFormat` if an mp3 file doesn't exists at the specified path.
+     */
+    func readFileFrom(path: String) throws -> Data {
         let validPath = URL(fileURLWithPath: path)
         guard validPath.pathExtension.caseInsensitiveCompare("mp3") == ComparisonResult.orderedSame else {
             throw ID3TagEditorError.invalidFileFormat
         }
+
         let mp3 = try Data(contentsOf: validPath)
         return mp3
     }
+
+    /**
+      Read only the ID3 header of the mp3 file at path
+
+      - parameter path: the path to the mp3 file
+
+      - returns: ID3 header data of the file
+
+      - throws: Could throw `InvalidFileFormat` if an mp3 file doesn't exists at the specified path, or if the file
+     does not contain the entire ID3 header
+     */
+    func readID3HeaderFrom(path: String) throws -> Data {
```

Comment:
This should be named something like `readID3TagFrom`, because it is the entire tag, not the header. Ok?